plenum/test/freshness/test_freshness_batch_updates_last_ordered.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+
+from plenum.common.messages.node_messages import PrePrepare
+from plenum.server.replica import Replica
+
+from plenum.common.constants import DOMAIN_LEDGER_ID
+from plenum.common.exceptions import PoolLedgerTimeoutException
+from plenum.server.node import Node
+from plenum.test.helper import freshness, sdk_send_random_and_check, primary_disconnection_time
+from plenum.test.spy_helpers import getSpecificDiscardedMsg
+from plenum.test.view_change.helper import restart_node
+
+# 20 secs, so node could restart
+FRESHNESS_TIMEOUT = 20
```

Comment:
I think we can make it less, like 5 secs, can't we?

Reply:
It is possible to lower it to 10 secs, but not less. Otherwise, view_change will be initiated because of freshness.